src/components/Toolbar/Toolbar.tsx

```diff
@@ -36,26 +37,27 @@ const Toolbar: FC<Props> = ({ onWidgetDrag }) => {
     [onWidgetDrag]
   );
 
-  return (
-    <Container>
-      <div
-        draggable
-        unselectable="on"
-        data-widget-type="visualization"
-        onDragStart={dragStartHandler}
-        onDragEnd={dragEndHandler}
-      >
-        <WidgetItem icon="bar-widget-vertical" text={t('widget_item.chart')} />
-      </div>
+  const item = (type: string, icon: IconType, text: string): JSX.Element => {
```

Comment:
please move `item` to a separate component

Reply:
@apryka your suggestions were applied.